src/handlers/tokens.js

```diff
@@ -0,0 +1,66 @@
+const Cache = require('node-cache');
+
+const CHUNK_SIZE = 25;
+
+const cache = new Cache({ stdTTL: 60 * 60, checkperiod: 0, useClones: false });
+
+function parseTokenNamesFromURI ({ tokens }) {
+    try {
+        const tokensCollection = decodeURIComponent(tokens).split(',');
+        if (!tokensCollection[0].length) {
+            throw new Error('Argument is not an array or being null');
+        }
+        return tokensCollection;
+    } catch (err) {
+        console.error(err);
+        return [];
+    }
+}
+
+function arrayPartition (tokens, size) {
```

Comment:
This function is generic and not tied to "tokens" per se.

So more generaic arg name would fit there: tokens => array